WotC_VestSlot/Config/XComVestSlot.ini

```diff
@@ -4,3 +4,22 @@
 ; the default behavior (all soldiers have vest slots) is disabled
 ; and only soldier with this abilities gain a vest slot
 ;+AbilityUnlocksVestSlot=AbilityTemplateName
+
++AllowedItemCategories = "defense"
+
+;	If this is set to "true", the Vest Slot will attempt to always be occupied 
+;	by an infinite item with a valid item category and the highest tier.
+;	Note: this does not apply to soldiers that you meet for the first time in Tactical, 
+;	such as soldiers you get on Gatecrasher.
+bDisallowEmpty = false
```

Comment:
Negating terms for bools are always somewhat confusing. e.g. `ShouldntDoSomething=false` 
Could we change the name of this bool for something thats worded positively like `bPrefillSlot` or `bAllowEmpty`?

Reply:
Absolutely, I wanted to make it a positive at first too, but then changed to a negative when I thought we might want to keep the VestSlot.ini unchanged, so bDisallowEmpty can function with "false" as the default value. We then decided we can make changes to the .ini file, but I kept the first working implementation cuz of railway thinking. Changed and pushed. 
I did have some weird behavior during testing, like I wasn't able to flip the config bool from "true" to "false" using my meta mod, but I think these kind of issues always crop up when using locally built mods.